src/controllers/commands/CommandController.cpp

```diff
@@ -482,6 +482,21 @@ void CommandController::initialize(Settings &, Paths &paths)
 
         QString userName = words[1];
         stripUserName(userName);
+
+        if (words.size() > 2)
+        {
+            QString channelName = words[2];
+            stripChannelName(channelName);
+
+            ChannelPtr channelTemp =
+                getApp()->twitch2->getChannelOrEmpty(channelName);
+
+            if (channelTemp->getType() != Channel::Type::None)
+            {
+                channel = channelTemp;
+            }
```

Comment:
Could there be some sort of a warning that Chatterino doesn't know about the supplied channel?

Reply:
Yeah, i though about it as well, but i am not sure where to show that warning...

Maybe a message in chat, similar to how it works when supplying invalid arguments?

![chatterino 08-15 21 48 37](https://user-images.githubusercontent.com/80157503/129490709-71631f8e-1e55-4d4c-aee4-83bf9833279f.png)

Also, this message should probably be updated in general to include the new argument.

Reply:
Ok, I added an error message using a system message displayed in chat, maybe take a quick look at it if it seems okay :)

Reply:
The error message works as expected, however now since you don't override `channel` with contents of `channelTemp`, the command always falls back to the current channel.

Reply:
Yeah i did that on purpose, would be a weird UX to display an error message but still go through with an action as if nothing happened imo

Could change it back though of course

Reply:
Oh yeah no wait i see the problem now FDM

Reply:
Should be fixed now

Reply:
Feel free to resolve this conversation :)